client/containers/problems_database.js

```diff
@@ -0,0 +1,32 @@
+export default [
+    {
+        id: '1',
+        userName: 'sasha99',
+        creationDate: '1.01.2017',
+        subject: 'history',
+        description: 'How many fighter jets?!',
+        rating: '27',
+        answersCount: '4',
+        isAnswered: true
+    },
+    {
+        id: '2',
+        userName: 'baRon',
+        creationDate: '2.02.2017',
+        subject: 'math',
+        description: '2 + 2 * 2 ?',
+        rating: '4',
+        answersCount: '18',
+        isAnswered: true
+    },
+    {
+        id: '3',
+        userName: 'Georg Marvin',
+        creationDate: '3.03.2017',
+        subject: 'javascript',
+        description: 'How can I ng-repeat in React?',
```

Comment:
You already did that using simple items.map(item => <ItemComponent item={item}/>)